external/jshint/jshint.js

```diff
@@ -58488,7 +58488,9 @@ var warnings = {
 	W101: i18n._("Line is too long."),
 	W102: i18n._("Trailing whitespace."),
 	W103: i18n._("The '{a}' property is deprecated."),
-	W104: i18n._("'{a}' is only available in JavaScript 1.7."),
+	W104: i18n._("It looks like you're trying to use '{a}'. '{a}' is a " +
+        "JavaScript 1.7 feature and is not currently supported in this " +
```

Comment:
@pamelafox I think that ES6 is the older new version while JavaScript 1.7 is still in progress. I did some looking around and let, const, and destructuring assignments are all 1.7 features, while pretty much everything else new is ES6 (see [1.7](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.7) and [ES6](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla)). I'm pretty sure this could be changed if you think it's necessary, the only reason I differentiated is because the original messages differentiated between ES6 and JavaScript 1.7.